File: MovieManiacs/Core/InfoModels/Movie.cs
using System;

namespace Core.InfoModels
{
    public class Movie
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public string Name { get; set; }
        public Nullable<int> Year { get; set; }
        public Nullable<System.DateTime> ReleaseDate { get; set; }
        public string ImageUrl { get; set; }
        public Nullable<int> Rank { get; set; }
        public string Genre { get; set; }
        public string Description { get; set; }
        public string Starring { get; set; }
    }
}

File: Android/MovieManiacs/app/src/main/java/bg/uni_sofia/fmi/moviemaniacs/models/Movie.java
package bg.uni_sofia.fmi.moviemaniacs.models;

public class Movie {

    public int Id;
    public float Rank;
    public String ReleaseDate;
    public int Year;
    public String description;
    public String genre;
    public String imageurl;
    public String starring;
    public String title;

    @Override
    public String toString() {
        return "Movie{" +
                "Id=" + Id +
                ", Rank=" + Rank +
                ", ReleaseDate='" + ReleaseDate + '\'' +
                ", Year=" + Year +
                ", description='" + description + '\'' +
                ", genre='" + genre + '\'' +
                ", imageurl='" + imageurl + '\'' +
                ", starring='" + starring + '\'' +
                ", title='" + title + '\'' +
                '}';
    }
}
